.github/workflows/linux-eic-shell.yml

```
@@ -874,6 +874,7 @@ jobs:
         uses: nickofthyme/object-remap@v2
         with:
           include.*.pr: ${{ toJSON(fromJSON(steps.query.outputs.data).*.number) }}
```

There is some automatic json to variable name going on in the remap action. Default is camelCase but we want snake_case.
```suggestion
          __case: snake
          include.*.pr: ${{ toJSON(fromJSON(steps.query.outputs.data).*.number) }}
```